ci-operator/config/openshift/cluster-api-provider-kubevirt/openshift-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -78,6 +101,242 @@ tests:
+        oc kustomize config/rbac | oc apply -f -
+
+        # Deploy webhook
+        oc kustomize config/webhook | oc apply -f -
+
+        # deploy the KAPK operator
+        cat <<EOF | oc apply -f -
+        apiVersion: apps/v1
+        kind: Deployment
+        metadata:
+          name: capi-kubevirt-operator
+          namespace: system
+        spec:
+          selector:
+            matchLabels:
+              name: cluster-api
+              app: cluster-api
+              hypershift.openshift.io/control-plane-component: cluster-api
+          replicas: 1
+          template:
+            metadata:
+              labels:
+                name: cluster-api
+                app: cluster-api
+                hypershift.openshift.io/control-plane-component: cluster-api
+                control-plane: controller-manager
+            spec:
+              terminationGracePeriodSeconds: 10
+              tolerations:
+              - key: node-role.kubernetes.io/master
+                effect: NoSchedule
+              volumes:
+              - name: capi-webhooks-tls
+                secret:
+                  secretName: capi-webhooks-tls
+                  defaultMode: 420
+              containers:
+              - name: manager
+                image: ${CAPK_IMAGE}
+                imagePullPolicy: Always
+                volumeMounts:
+                - name: capi-webhooks-tls
+                  readOnly: true
+                  mountPath: /tmp/k8s-webhook-server/serving-certs
+                env:
+                - name: MY_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                command:
+                - "/manager"
+                args:
+                - --namespace=\$(MY_NAMESPACE)
+                - --alsologtostderr
+                - --v=4
+                - --leader-elect=true
+                ports:
+                - name: healthz
+                  containerPort: 9440
+                  protocol: TCP
+                livenessProbe:
+                  httpGet:
+                    port: healthz
+                    path: /healthz
+                readinessProbe:
+                  httpGet:
+                    port: healthz
+                    path: /healthz
+              serviceAccountName: manager
+        EOF
+      dependencies:
+      - env: CAPK_IMAGE
+        name: cluster-api-kubevirt-controller
+      from: capk-oc-bin-image
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    - as: create-and-test-cluster
+      commands: |-
+        set -x
+        export NODE_VM_IMAGE_TEMPLATE=quay.io/kubevirtci/fedora-kubeadm:35
+        export IMAGE_REPO=k8s.gcr.io
+        export CRI_PATH="/var/run/crio/crio.sock"
+        clusterctl generate cluster kvcluster --kubernetes-version v1.21.0 --control-plane-machine-count=1 --worker-machine-count=1 --from templates/cluster-template.yaml | oc apply -f -
```

Comment:
I think we need to wait for the capk operator to come online. The CI lane fails because the webhook isn't serving yet.

so, wait for the operator deployment. probably something like this...

```
oc wait -n system --for=condition=Available=true deployment/capi-kubevirt-operator --timeout=10m
```


```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "validation.kubevirtmachinetemplate.infrastructure.cluster.x-k8s.io": Post "https://webhook-service.system.svc:443/validate-infrastructure-cluster-x-k8s-io-v1alpha1-kubevirtmachinetemplate?timeout=10s": no endpoints available for service "webhook-service"
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "validation.kubevirtmachinetemplate.infrastructure.cluster.x-k8s.io": Post "https://webhook-service.system.svc:443/validate-infrastructure-cluster-x-k8s-io-v1alpha1-kubevirtmachinetemplate?timeout=10s": no endpoints available for service "webhook-service"
```